src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -0,0 +1,20 @@
+import { Redirect, Route } from 'react-router-dom';
+import { useUser } from '../../hooks/useUser';
+
+export default function PrivateRoute({ children, ...routeProps }) {
+  const { user } = useUser();
```

Comment:
same here with an empty line - the `return` statement is it's own separate stanza in the poem of your code